src/osd/ReplicatedBackend.cc

```diff
@@ -1216,9 +1210,9 @@ void ReplicatedBackend::sub_op_modify_impl(OpRequestRef op)
   rm->localt.register_on_applied(
     parent->bless_context(
       new C_OSD_RepModifyApply(this, rm)));
-  list<ObjectStore::Transaction*> tls;
-  tls.push_back(&(rm->localt));
-  tls.push_back(&(rm->opt));
+  vector<ObjectStore::Transaction> tls;
```

Comment:
consider a `tls.reserve(2);` here to make sure we don't do separate allocations for the two `push_back()` calls below